tools/cell_census_builder/tests/test_builder.py

```diff
@@ -9,18 +10,15 @@
 import pyarrow as pa
 import pytest
 import tiledbsoma as soma
-from scipy.sparse import csc_matrix
-
-from api.python.cell_census.src import cell_census
-from tools.cell_census_builder.__main__ import build, make_experiment_builders
-from tools.cell_census_builder.datasets import Dataset
-from tools.cell_census_builder.globals import (
+from cell_census_builder.__main__ import build, make_experiment_builders
+from cell_census_builder.datasets import Dataset
+from cell_census_builder.globals import (
     CENSUS_DATA_NAME,
     CENSUS_INFO_NAME,
     CENSUS_X_LAYERS_PLATFORM_CONFIG,
 )
-from tools.cell_census_builder.mp import process_initializer
-from tools.cell_census_builder.util import uricat
+from cell_census_builder.mp import process_initializer
+from scipy.sparse import csc_matrix
```

Comment:
super tiny nit: I believe we use the Google python style, right?  So you should do `from scipy import sparse` and then spell out `sparse.csc_matrix`

[reference](https://google.github.io/styleguide/pyguide.html#22-imports)

Reply:
done